design-system/materials/internals/definition.yaml

```diff
@@ -326,6 +369,21 @@ decisionGroupsByTheme:
           description: ''
           choice: font-family-body
 
+    heights:
+      label: Height
+      prefix: height
+      decisions:
+        height-for-button-as-big:
+          choice: '32px'
```

Comment:
I'm using actual CSS values here as I don't see it makes sense to have a height scale since different components might have different needs so I guess we would end up with tokens like `height-32: 32px` and we would be adding new ones for every new use case.
However, if you think it would be better to have such a scale, I can refactor this part.

Reply:
I think it's fine this way, thanks 😊